spec/takeaway_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'takeaway'
+describe Order do
+  it 'should send menu' do
+    menu = MainMenu.new
+    expect(subject.set_menu).to eq(menu.dishes)
+  end
+  it 'should let you choose items' do
+    expect(Order.new('pizza, curry').items).to eq(('pizza, curry'))
+  end
+  it 'should confirm total' do
+    customer = Order.new('pizza, curry')
+    customer.set_menu
+    expect(customer.confirm_total).to eq("pizza = 5.0 curry = 2.2 total = £7.2")
+  end
+  it 'texts conformation through order' do
+    puts 'enter number'
+    number = gets.chomp
```

Comment:
The trouble here is that to run the test you need a human there typing in the answers, and the human needs to know what they're supposed to type in. Ideally, all the tests will run automatically, so the test suite can be done in seconds. We need to stub `gets` so that when `gets` is called, it returns something pre-defined in your test, instead of using the normal `gets` method.

Reply:
I just had this because i properly shouldn't push my phone number to GitHub